entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/impl/EntityRecordService.java

```diff
@@ -45,6 +45,12 @@
     public EntityRecord saveEntityRecord(EntityRecord er) {
 	return entityRecordRepository.save(er);
     }
+    
+    public EntityRecord disableEntityRecord(EntityRecord er) {
+    	er.setDisabled(true);
+		return saveEntityRecord(er);
```

Comment:
each database operation must ensure that the created timestamp exists, and the modified timestamp needs to be updated. We better include these updated in the saveEntityRecord method

Reply:
I do not understand this. We do not modify here the entity, just disable it, right? And it is already checked that the entity exists in the db.